spec/services/claims/fee_calculator/unit_price_spec.rb

```diff
@@ -0,0 +1,141 @@
+RSpec.configure do |c|
```

Comment:
**[not a blocker]** Any reason for this not to be within the scope of the `shared_examples` that require it, if the intent of this configuration is to apply to them only?

If I'm not mistaken, running the whole suite, with the random seed, all the specs will be loaded (including this one) first, which means this configuration will be active for all of them.